pkg/controllers/cloudmap_controller.go

```diff
@@ -70,6 +70,8 @@ func (r *CloudMapReconciler) Reconcile(ctx context.Context) error {
 		return err
 	}
 
+	//TODO: Fetch list of namespaces from Cloudmap and only reconcile the intersection
```

Comment:
I wonder what should happen to the service imports if we delete the namespace. [Issue](https://github.com/aws/aws-cloud-map-mcs-controller-for-k8s/issues/83)

Reply:
Delete the namespace in the cluster or cloudmap?